nss-day-cohort-39/nutshell-veridian-vicuna
# Requesting to merge the rendered tasks to master

### Changes
- Added all necessary files for the list of tasks
- Got the tasks to only show if they relate to the current user
- Got the task list rendered on the DOM right next to the events list
- Added the delete button but haven't gotten it to work yet

### Testing
- Make tasks that have your user id tied into them
- Make sure that you can see your list of tasks on the DOM in the task list section
- Make sure that there is a delete button (It is not supposed to work yet)
